### PR TITLE
Setting 7pm local as the time on the due date

### LINKED ISSUE #324: SCSB renewal due date not being calculated correctly
Items for faculty and grad students (where the due date is June 15th of the next year) are being calculated as June 14th at 11:59 PM. I am wondering if this is due to the time-zone difference (June 15th at midnight in UTC is June 14th in EST). Is this due to what I think it is, or is there some other thing at play?

### PR DESCRIPTION
**Pull request by carolyncole**
fixes #324

### CODE REVIEW

**Comment by carolyncole** [comment]
I ran this PR from staging and I believe it is working.  If you open production alma and search on barcode for `32044002960052`  You will see it has a dat of 6/15.  If you look at `32044031629892` you will see it has a date of 6/14

**Review by maxkadel — COMMENTED** [review]
I think what's confusing about this is that it'll go back to 23:59:59 when it gets to alma - is there a way to make it clearer that Alma is going to ignore the time, but it needs the time in order to parse the info correctly? (I'm thinking forward to a time when I don't know why the due date in Alma isn't 7 pm EST)

**Review by kevinreiss — APPROVED** [review]

File: app/models/alma_renew/item.rb [hunk 1]
```diff
@@ -78,7 +78,9 @@ def ncip_renew_body(due_date:)
             xml['ns1'].ItemId do
               xml['ns1'].ItemIdentifierValue item_barcode
             end
-            xml['ns1'].DesiredDateDue due_date
+            # setting the time to 7pm so that Alma does not revert the time to 11:59 the day before
```
Comment (carolyncole) [inline app/models/alma_renew/item.rb:81]: @maxkadel Make sense?

Comment (maxkadel) [reply app/models/alma_renew/item.rb:81]: Looks good! 🎉

**Pull request merged**